docker/mainnet/mainnet-container.env

```diff
@@ -0,0 +1,22 @@
+## These variables should be defined by the user before configuring/running a node 
+
+## MAINNET VERSION
+
+# syntax is <ip>:26656 or <dns_address>:26656
+P2P_EXTERNAL_ADDRESS=
+
+# moniker is a friendly name for your node
+NODE_MONIKER=
+
+# 
+CHEQD_VOLUME_SOURCE=
+
+## Default parameters than can be changed to custom values if desired
+
+NETWORK=mainnet
+FASTSYNC_VERSION="v0"
+MIN_GAS_PRICES="25ncheq"
+RPC_LADDR=tcp://0.0.0.0:26657
+P2P_MAX_PACKET_MSG_PAYLOAD_SIZE=10240
+P2P_RECV_RATE=20000000
+P2P_SEND_RATE=20000000
```

Comment:
This list should contain **all** the possible variables that are used in node-runner.sh, except for P2P_SEEDS (since that's downloaded and set automatically from GitHub)